src/api.ts

```diff
@@ -47,10 +47,13 @@ router.post('/*', async (req, res) => {
     const result: any = await serve(key, getData, [url, query, key, caching]);
     if (result.errors) {
       capture(new Error('GraphQl error'), result.errors);
-      return res.status(500).json(result);
+      return res.status(400).json(result);
     }
     return res.json(result);
   } catch (error: any) {
+    if (error.errors) {
+      return res.status(500).json(error);
+    }
```

Comment:
Fetch error can contain `errors`? 🤔

Reply:
This is for errors returned by graphql, like "Skip limit can not be over X", formatted like `{errors: [{message: 'xxx'}]}`

Reply:
These go into `catch`? 🤔  i don't think so

Reply:
From here https://github.com/snapshot-labs/subgrapher/blob/main/src/api.ts#L48

This function is not used exclusively on try/catch